R/profile_plot.R

```diff
@@ -163,11 +171,13 @@ profile_plot <- function(mydir, rep, para, profilesummary) {
   # parameter vs. final depletion
   plot(x, depl, type = "l", lwd = 2, xlab = label, ylab = "Fraction of unfished", ylim = c(0, 1.2))
   points(est, depl_est, pch = 21, col = "black", bg = "blue", cex = 1.5)
-  abline(h = c(thresh, btarg), lty = c(2, 2), col = c("red", "darkgreen"))
-  legend("bottomright",
-    legend = c("Management target", "Minimum stock size threshold"),
-    lty = 2, col = c("red", "darkgreen"), bty = "n"
-  )
+  abline(h = c(btarg, thresh), lty = c(2, 2), col = c("red", "darkgreen"))
+  if(btarg > 0){
+    legend("bottomright",
+           legend = c("Management target", "Minimum stock size threshold"),
```

Comment:
Indenting two spaces is more common.

Reply:
I think this package and all others in `pfmc-assessments` would benefit from adding github actions to deal with this stuff, specifically:
* `ghactions4r::use_spell_check()`
* `ghactions4r::use_update_roxygen_docs()`
* `ghactions4r::use_style_r_code()`

The 2nd and 3rd on that list can be replaced by `ghactions4r::use_doc_and_style_r(use_rm_dollar_sign = TRUE)` if you want the additional change of converting things like `x$pars$Label` to `x[["pars"]][["Label"]]` (which I think makes the code harder to read but is apparently good practice because it avoids bugs caused by partial matching).